src/System/Boot.tsx

```diff
@@ -81,25 +82,27 @@ export const Boot = track(undefined, {
               <EnvironmentProvider environment={props.relayEnvironment}>
                 <ErrorBoundary>
                   <MediaContextProvider onlyMatch={onlyMatchMediaQueries}>
-                    <ToastsProvider>
-                      <StickyProvider>
-                        <AuthIntentProvider>
-                          <AuthDialogProvider>
-                            <DismissibleProvider
-                              userID={props.user?.id}
-                              keys={PROGRESSIVE_ONBOARDING_KEYS}
-                            >
-                              <CookieConsentManager>
-                                <FocusVisible />
-                                <SiftContainer />
+                    <GlobalMeProvider>
```

Comment:
If we go with this route, I wondered if there is a better place to load this data than its own provider. `SystemContextProvider` has `user`, but it is above the relay environment so not so easy to query.